mkdocs.yml

```diff
@@ -74,6 +74,7 @@ nav:
           - 'Trade Book': 'member/Books/tradebook.md'
           - 'Fund Book': 'member/Books/fundbook.md'
       - 'Broking Details': 'member/broking-details.md'
+      - 'Fixed Income': 'member/fixed-income.md'
```

Comment:
why this change?

Reply:
This is for the Fixed Income page though the page is not pushed yet so it won't show on the site

Reply:
Since we want to add that page to the help site anyways, I thought of adding it

Reply:
Ok